res/graphics/quest icons/wheelchair_outside.svg

```diff
@@ -0,0 +1 @@
+/home/flo/www/StreetComplete/res/graphics/quest icons/toilets_wheelchair.svg
```

Comment:
The change wasn't wrong, I've symlinked `wheelchair_outside.svg` to `toilets_wheelchair.svg`, as the former had no separate icon. However, this is not needed anymore (because I dropped the icon information), so I reverted the commit.

The rename of `guidepost.svg` to `board_type.svg` because the corresponding Android drawable is also called `ic_quest_board_type` and not `ic_quest_guidepost`: https://github.com/streetcomplete/StreetComplete/blob/86b082850c0ea13c0a942255edd7ce5a94808460/app/src/main/java/de/westnordost/streetcomplete/quests/board_type/AddBoardType.kt#L16

Now, all quest icons' drawable names match the SVG file names.